portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -1,56 +1,61 @@
 // Client ID and API key from the Developer Console
-const CLIENT_ID =
+const CLIENT_ID_SHEETS =
```

Comment:
Consider using SHEETS as the prefix, rather than suffix.

SHEETS_CLIENT_ID
SHEETS_API_KEY
...

Reply:
Done.